### PR TITLE
Further RU update

### PR DESCRIPTION
**Pull request by paladinkolovrat**

### CODE REVIEW

**Comment by jixxed** [comment]
This PR has a conflict, 1 name was changed

**Comment by paladinkolovrat** [comment]
Fixed, I think

File: application/src/main/resources/locale/blueprint/horizons/names.csv [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
+blueprint.horizons.name.anti_corrosion_cargo_rack,Anti Corrosion Cargo Rack,Anti Corrosion Cargo Rack,,,,,Антикоррозийный грузовой стеллаж
 blueprint.horizons.name.armour,Armour,Armour,Rüstung,Armadura,Armure,Armaduras,Переборки
 blueprint.horizons.name.auto_field_maintenance_unit,Auto Field Maintenance Unit,Auto Field Maintenance Unit,Automatische Feldwartungs-Einheit,Unidad de mantenimiento de campo automático,Unité de maintenance des champs automobiles,Unidade de manutenção de campo automático,Блок автоматического полевого ремонта
 blueprint.horizons.name.ax_explosive_munitions,AX Explosive Munitions,AX Explosive Munitions,AX Explosivmunition,Municiones explosivas de hacha,Munitions explosives à hache,Munições explosivas do machado,Взрывчатые боеприпасы АИ
-blueprint.horizons.name.ax_missile_rack,Ax Missile Rack,Ax Missile Rack,,,,,
-blueprint.horizons.name.ax_missile_rack_pre,Ax Missile Rack,Ax Missile Rack,,,,,
-blueprint.horizons.name.ax_multi_cannon,Ax Multi Cannon,Ax Multi Cannon,,,,,
-blueprint.horizons.name.ax_multi_cannon_pre,Enhanced Ax Multi Cannon,Enhanced Ax Multi Cannon,,,,,
```
[the remark this answers is not in the document]
Comment (jixxed) [reply application/src/main/resources/locale/blueprint/horizons/names.csv:16]: Enhanced Ax Multi Cannon

File: application/src/main/resources/locale/blueprint/horizons/names.csv [hunk 2]
```diff
@@ -64,160 +64,160 @@ blueprint.horizons.name.engineer_e2a,"    E2A. Broo Tarquin",,,,,,"    E2A. Бр
...
-blueprint.horizons.name.fighter_hangar,Fighter Hangar,Fighter Hangar,,,,,
-blueprint.horizons.name.first_class_passenger_cabin,First Class Passenger Cabin,,,,,,
-blueprint.horizons.name.first_class_passenger_cabin.short,(FIR),,,,,,
+blueprint.horizons.name.festive_launcher,Festive Launcher,Festive Launcher,,,,,Праздничная зенитная установка
+blueprint.horizons.name.festive_launcher.green,Festive Launcher Green,Festive Launcher Green,,,,,Праздничная зенитная установка (зеленая)
+blueprint.horizons.name.festive_launcher.red,Festive Launcher Red,Festive Launcher Red,,,,,Праздничная зенитная установка (красная)
+blueprint.horizons.name.festive_launcher.yellow,Festive Launcher Yellow,Festive Launcher Yellow,,,,,Праздничная зенитная установка (желтая)
+blueprint.horizons.name.fighter_hangar,Fighter Hangar,Fighter Hangar,,,,,Ангар для истребителя
+blueprint.horizons.name.first_class_passenger_cabin,First Class Passenger Cabin,,,,,,Пассажирские каюты первого класса
+blueprint.horizons.name.first_class_passenger_cabin.short,(ПРВ),,,,,,
```
[the remark this answers is not in the document]
Comment (paladinkolovrat) [reply application/src/main/resources/locale/blueprint/horizons/names.csv]: I hope it is fixed this time

**Pull request merged**